doc/source/getting_started.rst

```diff
@@ -0,0 +1,99 @@
+.. _getting_started:
+
+***************
+Getting started
+***************
+
+Installation
+------------
+
+.. raw:: html
+
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 d-flex install-block">
+                <div class="card install-card shadow w-100">
+                <div class="card-header">
```

Comment:
A note here: I have a PR in pandas to replace this raw html with a nice sphinx extension to do the same (`sphinx-panels`), see https://github.com/pandas-dev/pandas/pull/34065. You might want to do the same here.

(and the same comment for the cards in index.rst)

Reply:
Oh great! Because I am not a fan of this mixing. Although I in the following PR I added the ability to click on the icons. Does the extension do this?

Reply:
Yes, sphinx-panels (actually bootstrap) has a "stretched-link" class that makes the full card clickable